appeals/web/src/server/lib/mappers/appellantCase.mapper.js

```diff
@@ -29,348 +31,151 @@ export function initialiseAndMapData(appellantCaseData, appealDetails, currentRo
 	currentRoute =
 		currentRoute[currentRoute.length - 1] === '/' ? currentRoute.slice(0, -1) : currentRoute;
 
+	/**
+	 * Generates a document field with defaults
+	 *
+	 * @param {Object} options
+	 * @param {string} options.id
+	 * @param {string} options.text
+	 * @param {FolderInfo|null|undefined} options.folderInfo
+	 * @param {string} [options.cypressDataName]
+	 * @returns {Instructions}
+	 */
+	const documentField = ({ id, text, folderInfo, cypressDataName }) => {
```

Comment:
This name feels a bit vague - the term "field" is too ambiguous in my opinion. Maybe "instruction" would be a better fit?

Reply:
OK, we can go with `documentInstruction`